.github/scripts/install_dependencies.sh

```diff
@@ -31,31 +31,28 @@ sudo apt install -y \
   libxft-dev \
   libxml++2.6-dev \
   libreadline-dev \
-  tcl-dev \
+  tcllib \
+  tcl8.6-dev \
```

Comment:
Regarding tcl errors, try to include the following packages

- https://packages.ubuntu.com/jammy/libtcl8.6